utils/build-script-impl

```diff
@@ -2265,17 +2440,17 @@ for deployment_target in "${HOST_TARGET}" "${CROSS_COMPILE_TOOLS_DEPLOYMENT_TARG
                 if [[ -z "${INSTALL_LLDB}" ]] ; then
                     continue
                 fi
-                case "$(uname -s)" in
-                    Linux)
+                case ${host} in
+                    linux-*)
                         ;;
-                    FreeBSD)
+                    freebsd-*)
                         ;;
-                    CYGWIN_NT-10.0)
+                    cygwin-*)
                         ;;
-                    Darwin)
+                    macosx-*)
                         set_lldb_build_mode
                         pushd ${LLDB_SOURCE_DIR}
-                        xcodebuild -target toolchain -configuration ${LLDB_BUILD_MODE} install ${lldb_xcodebuild_options[@]} DSTROOT="${INSTALL_DESTDIR}" LLDB_TOOLCHAIN_PREFIX="${TOOLCHAIN_PREFIX}"
+                        xcodebuild -target toolchain -configuration ${LLDB_BUILD_MODE} install ${lldb_xcodebuild_options[@]} DSTROOT="${host_install_destdir}"
```

Comment:
LLDB shouldn't be a problem, either - we're setting the `DSTROOT` parameter when calling `xcodebuild`, which is where the products will be placed when installing. Not passing the prefix parameter shouldn't be a problem.